kiali.go

```diff
@@ -103,9 +104,9 @@ func main() {
 	// we need first discover Jaeger
 	if config.Get().ExternalServices.Tracing.Enabled {
 		status.DiscoverJaeger()
-		_, err := business.GetServices()
+		_, err := business.GetJaegerServices()
 		if err != nil {
-			business.JaegerAvailable = false
+			appstate.JaegerAvailable = false
```

Comment:
@aljesusg I think we should improve this , if I understand correctly here, when Jaeger cannot be reached at Kiali startup, it's definitely turned off. One would have to restart Kiali to get this `JaegerAvailable` flag to true again.
This happened to me while testing, the call to Jaeger ended in timeout [1] so `JaegerAvailable` was set to false even if it was available again shortly after.
Maybe this check to jaeger availability shouldn't be done just once at startup, WDYT?

[1] logs: `E0529 09:17:05.520601       1 kiali.go:109] Jaeger is not available : Get http://tracing/jaeger/api/services:  net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)`

Reply:
This is why I proposed about the refresh endpoint to avoid restart pod and reset the discover, I talked about this with @lucasponce .

Reply:
Cool, ok!
Is there already a JIRA?

Reply:
no, because we decided not do it (Discussion: https://github.com/kiali/kiali/pull/1022/files#r277584377 You were in discussion ) , but I am +1 in do it..... @lucasponce  ?? ^^.

I think that the problem here is when there are more than 1 pod and you want to refresh...

Reply:
That discussion was about the `JaegerEnabled` one, not `JaegerAvailable`.
I think the case with `JaegerAvailable` is much worst because it depends on Kiali being able to reach Jaeger at startup, so it makes it very dependant on the startup sequence of the services.

Reply:
JIRA created: https://issues.jboss.org/browse/KIALI-2964